devbox.json

```diff
@@ -25,7 +25,7 @@
     "llvmPackages_14.clangUseLLVM@14.0.6",
```

Comment:
Do you wanna touch up some of the tooling versions while we're at it?

Reply:
Sure - I wasn't sure re: backporting other tooling updates but I'll just check for each branch.

Reply:
I'll ship this and do another PR for various tooling updates. Makes sense to keep it atomic :)